experimental/displaynames/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+//! [`icu_displaynames`](crate) is one of the ['ICU4X`] components.
+//!
+//! This API provides a mapping of locale display names for regions.
+//!
+//! <div class="stab unstable">
+//! 🚧 This code is experimental; it may change at any time, in breaking or non-breaking ways,
+//! including in SemVer minor releases. It can be enabled with the "experimental" feature
+//! of the icu meta-crate. Use with caution.
+//! </div>
+//!
+//! TODO: expand documentation
```

Comment:
take this out of the doc comment into a normal comment

Reply:
Done.